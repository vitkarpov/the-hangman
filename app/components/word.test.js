const Word = require("./word");

describe("Word", () => {
  it("renders correctly", () => {
    const currWord = ["h", "_", "_", "l", "o"];
    expect(new Word({ currWord }).render()).toMatchSnapshot();
  });
});
