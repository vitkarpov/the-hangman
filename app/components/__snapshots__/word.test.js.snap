// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Word renders correctly 1`] = `
"
      <div class=\\"word\\">
        <div class=\\"word-item\\">h</div><div class=\\"word-item\\">_</div><div class=\\"word-item\\">_</div><div class=\\"word-item\\">l</div><div class=\\"word-item\\">o</div>
      </div>
    "
`;
